tests/unit_tests/pydantic/utils.py

```diff
@@ -93,6 +93,8 @@ def load_geoips_yaml_plugin(interface_name, plugin_name):
     plugin via <interface>.get_plugin, this sometimes will cause errors due to different
     validation protocols.
 
+    TODO: Turn on a flag that just disables validation in 'get_plugin'
```

Comment:
We are not going to perform validation at run time,  correct (or disable with a flag?) ?  Assume all plug-ins have been pre validated?

Reply:
I think that we do intend to validate at run time. It's not an expensive operation and will ensure that nothing got edited incorrectly or corrupted. We will also validate at registry creation time, but I don't see that it hurts to validate twice. We do that currently using the jsonschema-based validation.